src/hotspot/share/runtime/thread.hpp

```diff
@@ -621,11 +621,6 @@ class Thread: public ThreadShadow {
 
   bool is_trace_suspend()               { return (_suspend_flags & _trace_flag) != 0; }
 
-  // VM operation support
-  int vm_operation_ticket()                      { return ++_vm_operation_started_count; }
```

Comment:
If this goes, then the declaration of `_vm_operation_started_count` can also go?

Reply:
Fixing